pages/lightning-extensions/faraday.en.mdx

```diff
@@ -3,80 +3,80 @@ import Callout from "nextra-theme-docs/callout";
 # Faraday
 
 <Callout emoji="&#x2139;">
-	Tijd: 5 minuten
+	Time: 5 minutes
 </Callout>
 
-Faraday is een [boekhoud tool](https://lightning.engineering/posts/2020-09-15-faraday-accounting) voor Lightning en on-chain. Handig, want je bent altijd wel wat sats kwijt aan dingen als transactiekosten die vaak onopgemerkt gaan. Het biedt onder andere de mogelijkheid om een uitdraai te maken naar CSV formaat.
+Faraday is an [accounting tool](https://lightning.engineering/posts/2020-09-15-faraday-accounting) for Lightning and on-chain. Useful, because you're always losing sats on things like transaction fees that often go unnoticed. Among other things, it offers the possibility to create a printout to CSV format.
 
-## Benodigdheden
+## Requirements
 
-* [Golang](/raspberry-pi/common-dependencies#golang) **\(update naar de nieuwste versie voordat je begint\)**
+* [Golang](/raspberry-pi/common-dependencies#golang) **(update to the latest version before you start)**.
```

Comment:
Hier rendert opeens een `#` in de lokale omgeving. Weet niet precies waar die vandaan komt :sweat_smile:

Reply:
ook maar gewoon escapen dan